src/TimeTrackerWindow.cpp

```diff
@@ -142,6 +145,20 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
 		}
 		break;
 	}
+	case MENU_APP_DUPLICATE_TASK:
```

Comment:
Can you check if more than one item is selected and if so, duplicate all of them?

Reply:
Done! 😄

Reply:
Now item 0 will always get duplicated, too. :)

Reply:
Hehe. I'll fix that 👍